### PR TITLE
Lejit app : Dashboard UI files

### PR DESCRIPTION
**Pull request by GoutamShanbhag**
@yogendrakvr 

Lejit app : Added UI files for the dashboard along with required styling and translation files

Thank you

### CODE REVIEW

File: React-Template-master/src/components/Appbar.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                            display: 'flex',
+                            alignItems: 'center'
+                        }}>
+                        <Box
+                            sx={{
+                                display: 'flex',
+                                width: '40%',
+                                alignItems: 'center'
+                            }}>
+                            <Tooltip title="Current User">
```
Comment (yogendrakvr) [inline React-Template-master/src/components/Appbar.tsx]: What is the use of this component ?

File: React-Template-master/src/components/Appbar.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                                <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                                    <IconButton sx={{ color: PURPLE.lighter }}>
+                                        <Avatar
+                                            sx={{
+                                                width: '48px',
+                                                height: '48px',
+                                                backgroundColor: PURPLE.lighter
+                                            }}>
+                                            <Typography variant="body2" sx={{ color: PURPLE.dark }}>
+                                                RS
```
Comment (yogendrakvr) [inline React-Template-master/src/components/Appbar.tsx]: Why is this hard-coded ?

File: React-Template-master/src/components/Appbar.tsx [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+                                            </Typography>
+                                        </Avatar>
+                                    </IconButton>
+
+                                    <Typography variant="body2">Rohit Sharma</Typography>
```
Comment (yogendrakvr) [inline React-Template-master/src/components/Appbar.tsx]: Why is this hard-coded ?

File: React-Template-master/src/i18n/config.ts [hunk 4]
```diff
@@ -1,16 +1,22 @@
...
+        fr: { translation: fnTranslation }
+
     }
 });
 
 export type SupportedLanguage = 'en' | 'fr';
-// Function to change lanuguage
+
+// Function to change language
+
```
Comment (yogendrakvr) [inline React-Template-master/src/i18n/config.ts:19]: Remove this line.

File: React-Template-master/src/i18n/fr/translation.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (yogendrakvr) [inline React-Template-master/src/i18n/fr/translation.json:1]: Make sure all the keys in the english and french are present in both the files

File: React-Template-master/src/theme/palette.ts [hunk 6]
```diff
@@ -66,10 +66,8 @@ const ERROR: PaletteColorOptions = {
     contrastText: '#fff'
 };
 
-export const BUTTONCOLORS: SimplePaletteColorOptions = {
-    light: '#EBEBF0',
-    dark: '#3E0EC3',
-    main: ''
+export const WHITE = {
```
Comment (yogendrakvr) [inline React-Template-master/src/theme/palette.ts]: There is no need to define white color as it is already present in the `common`. See line 87

**Review by yogendrakvr — CHANGES_REQUESTED** [review]

File: React-Template-master/src/theme/palette.ts (hunk 6 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/theme/palette.ts]: will fix this

File: React-Template-master/src/components/Appbar.tsx (hunk 3 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/components/Appbar.tsx]: i thought i will replace this with original data once Firebase is properly connected

File: React-Template-master/src/components/Appbar.tsx (hunk 1 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/components/Appbar.tsx]: i will remove it

File: React-Template-master/src/components/Appbar.tsx (hunk 2 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/components/Appbar.tsx]: i have added TODOs

File: React-Template-master/src/components/Appbar.tsx (hunk 3 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/components/Appbar.tsx]: i have added TODOs

File: React-Template-master/src/i18n/config.ts (hunk 4 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/i18n/config.ts:19]: okay

File: React-Template-master/src/i18n/fr/translation.json (hunk 5 above)
Comment (GoutamShanbhag) [reply React-Template-master/src/i18n/fr/translation.json:1]: okay

File: React-Template-master/src/theme/palette.ts [hunk 7]
```diff
@@ -80,6 +74,11 @@ export const NEUTRAL = {
     main: '#28293D',
     contrastText: '#fff'
 };
+
+export const PURPLE = {
```
Comment (yogendrakvr) [inline React-Template-master/src/theme/palette.ts:78]: What is the use of this color ?

**Review by yogendrakvr — CHANGES_REQUESTED** [review]

File: React-Template-master/src/i18n/config.ts (hunk 4 above)
Comment (yogendrakvr) [reply React-Template-master/src/i18n/config.ts:19]: This change is not added

**Review by yogendrakvr — APPROVED** [review]

**Pull request merged**